spec/system/candidate_interface/continuous_applications/submitting/candidate_tries_to_submit_an_application_for_a_course_with_closed_application_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate tries to submit an application choice which the course is closed' do
+  include SignInHelper
+  include CandidateSubmissionHelper
+  before do
+    given_i_am_signed_in
+    and_i_have_one_application_in_draft
+  end
+
+  scenario 'Apply is open but course not open for applications', time: mid_cycle(CycleTimetable.current_year) do
```

Comment:
Can we use `CycleTimetableHelper.mid_cycle` to keep all `time: ` options consistent?

Reply:
Done